### PR TITLE
[IMP] Project: add JS tests for project widgets

### PR DESCRIPTION
**Pull request by bastvdn**
Added JS tests for the following widgets:
- status_with_color (project.project, project.update)
- name_with_subtask_count (project.task)
- priority_switch (project.task)

Task-3660956

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/static/tests/project_status_with_color_selection_tests.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                },
+            },
+            arch: `
+                <kanban  class="o_kanban_test">
+                    <field name="status"/>
+                    <field name="id"/>
+                    <template>
+                        <t t-name="kanban-box">
+                            <div>
+                            <field name="status" widget="status_with_color" readonly="1" status_label="test status label"/>
```
Comment (pkyriakou) [inline addons/project/static/tests/project_status_with_color_selection_tests.js:43]: indentation :nerd_face:

File: addons/project/static/tests/project_status_with_color_selection_tests.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/** @odoo-module */
+
+import { getFixture } from "@web/../tests/helpers/utils";
+import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+
+let makeViewParams, target;
+
+QUnit.module("Project", (hooks) => {
+        hooks.beforeEach(() => {
+        makeViewParams = {
```
Comment (pkyriakou) [inline addons/project/static/tests/project_status_with_color_selection_tests.js:10]: indentation :neckbeard:

File: addons/project/static/tests/project_task_priority_switch_tests.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+/* @odoo-module */
+
+import { getFixture, triggerHotkey } from "@web/../tests/helpers/utils";
+import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+
+let makeViewParams, target;
+
+QUnit.module("Project", (hooks) => {
+        hooks.beforeEach(() => {
+        makeViewParams = {
```
Comment (pkyriakou) [inline addons/project/static/tests/project_task_priority_switch_tests.js:10]: indentation :thinking:

**Review by pkyriakou — COMMENTED** [review]
Thank you @bastvdn for your work :100: 
Left some very minor comments but I will green the task regardless :green_circle: :partying_face:

**Review by xavierbol — APPROVED** [review]
Thanks a lot! 🤩 💪🏼 

LGTM! 💪🏼 

robodoo r+

**Comment by xavierbol** [comment]
No worries, normally the runbot will be happy itself. 🤞🏼

**Pull request closed without merging**